scipy/stats/_continuous_distns.py

```diff
@@ -8745,6 +8756,15 @@ def isf_right(q, a, b):
 
         return out
 
+    def _entropy(self, a, b):
+        A = sc.digamma(a)
+        B = sc.digamma(b)
+        Z = B - A
+        C = np.log(np.sqrt(2 * np.pi * np.e) * Z)
+        D = (a * A - b * B) / (2 * Z)
+        h = C + D
+        return h
```

Comment:
```suggestion
        A = sc.ndtr(a)
        B = sc.ndtr(b)
        Z = B - A
        C = np.log(np.sqrt(2 * np.pi * np.e) * Z)
        D = (a * _norm_pdf(a) - b * _norm_pdf(b)) / (2 * Z)
        h = C + D
        return h
```

Reply:
This code doesn't work.

Reply:
This code doesn't work.

Reply:
Thanks, I didn't realize I misread the definition.